src/modules/service-catalog/components/service-catalog-list/ServiceCatalogListItem.tsx

```diff
@@ -66,7 +66,9 @@ const ServiceCatalogListItem = ({
       <ItemThumbnail size="medium" url={serviceItem.thumbnail_url} />
       <TextContainer>
         <ItemTitle>{serviceItem.name}</ItemTitle>
-        <ItemDescription>{serviceItem.description}</ItemDescription>
+        <ItemDescription
+          dangerouslySetInnerHTML={{ __html: serviceItem.description }}
```

Comment:
If you update [this test](https://github.com/zendesk/copenhagen_theme/blob/master/src/modules/service-catalog/components/service-catalog-list/ServiceCatalogListItem.spec.tsx#L31) to add some HTML tags in the description, you will cover this line.